src/modules/jobs/controllers/pokemon.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller, Get, Query } from '@nestjs/common';
+
+import { PokemonService } from '../services/pokemon.service';
+
+@Controller('/pokemon')
+export class PokemonController {
+  constructor(private readonly pokemonService: PokemonService) {}
+
+  /**
+   * 
+   * @param {string} id - string representation of a specific pokemon's id number
+   * @param {string} name - the name of a pokemon
+   * @returns {boolean} - a JSON object that indicates failure on a bad request or the pokemon data on a successful request
+   */
+  @Get(['/'])
+  async getSpecificPokemon(@Query('id') id: string, @Query('name') name: string) {
+    await new Promise(resolve => setTimeout(resolve, 500)); // just adds a delay for feel so request isn't instantaneous.
+    
+
+    if (!this.pokemonService.isGenOne(id, name)) {
+      return {
+        'flowStatus': 'FAILURE',
+        'flowStatusMessage': `${id ? 'Invalid id - must be between 1 and 151' : 'Invalid name - name must be a gen one pokemon'} `,
+      }
+    }
+
+    const pokemonData = await this.pokemonService.getPokemon(id || name);
+    
+    return {
+      'flowStatus': 'SUCCESS',
+      'flowStatusMessage': '',
```

Comment:
Let's add a flowStatusMessage. Sorry if I didn't elsewhere but we should. We can ask @tyler about what exactly it's used for